card/lib/card/cache.rb

```diff
@@ -40,6 +39,11 @@ def renew
         reset_local
       end
 
+      def prefix_root
+        @@prefix_root ||= ( cfg = Cardio.config and cfg = cfg.database_configuration and
+                            cfg[Rails.env]['database'] )
+      end
+
```

Comment:
fyi: looks like there are still about 60 results for "Rails" in card.  We got rid of the require, but autonomy is a ways off yet.